core/data/nlu.en.md

```diff
@@ -25,4 +25,36 @@
 - I received 1234
 - 1234
 - the code is 1234
-- it's 1234
\ No newline at end of file
+- it's 1234
+
+## intent: change_phone
+- change number
+- wrong number
+- change phone number
+- wrong phone number
+- replace number
+- replace phone number
+- i made a mistake
+- not the right number
+- not the right phone number
+
+## intent: did_not_get_code
+
+- no code
+- no validation code
+- didn't receive a code
+- didn't receive a validation code
+- didn't get a code
+- didn't get a validation code
+- did not receive a code
+- did not receive a validation code
+- did not get a code
+- did not get a validation code
+- haven't received a code
+- haven't received a validation code
+- haven't gotten a code
+- haven't gotten a validation code
+- didn't receive it
+- didn't get it
+- received nothing
+- got nothing
```

Comment:
We will need to clean up the NLU data later, some sentences are too generic to be part of such a specific intent.